scripts/update_downstream_ci.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+# This script will, for each Pkl repo, bump the version of `pkl.impl.ghactions` to the latest version, and
+# create a GitHub pull request.
+# It assumes that every repo exists in a forked account.
+#
+# Usage: ./update_downstream_ci.sh <your_github_account_name>
+
+set -eo pipefail
+
+MY_GIT_USER="$1"
```

Comment:
Can this be queried from `gh`?

Reply:
Ah, looks like we can: `gh api user --jq '.login'`